ohapi/tests/test_get_page.py

```diff
@@ -0,0 +1,33 @@
+import os
+import vcr
+import unittest
+import ohapi.api as api
+
+my_vcr = vcr.VCR(path_transformer=vcr.VCR.ensure_suffix('.yaml'),
+                 cassette_library_dir='ohapi/cassettes',
+                 filter_query_parameters=['access_token'])
+
+
+class test_get_page(unittest.TestCase):
```

Comment:
I'd prefer for `test_api.py` to have all tests relevant to `api.py`. Could you please move this class to that file?

Also, I'd like the class to be renamed to conform to PEP8 (https://www.python.org/dev/peps/pep-0008/#class-names), e.g. `APITestGetPage`